docs/content/preview/troubleshoot/cluster/replace_failed_peers.md

```diff
@@ -23,6 +22,23 @@ Assume you have a cluster where the following applies:
 
 These are the steps to follow:
 
+- Delete the tablet from tablet data from the broken peers if necessary, by running:
+
+```
+yb-ts-cli --server_address=NODE_BAD1 delete_tablet TABLET1
+yb-ts-cli --server_address=NODE_BAD2 delete_tablet TABLET1
+```
+
+- Trigger a remote bootstrap of `TABLET1` from `NODE_GOOD` to `NODE_BAD`.
+
+```
+yb-ts-cli --server_address=NODE_BAD1 remote_bootstrap NODE_GOOD TABLET1
+```
+
+Once the remote bootstrap finishes, `NODE_BAD2` should be automatically fixed and removed from its quorum, as it has gotten a majority of healthy peers.
```

Comment:
fixed? and yet removed from the quorum? or once there is a majority of healthy peers, it is somehow brought back online? This could be clearer...

Reply:
Rephrased to clarify that `NODE_BAD2` is removed from the quorum and `TABLET1` is fixed.